roles/create-devstack-local-conf/tasks/main.yml

```diff
@@ -122,7 +122,7 @@
       set -e
       set -x
       cat << EOF >> /tmp/dg-local.conf
-      enable_service neutron-fwaas-v1
+      enable_service q-agt q-svc q-dhcp q-l3 q-meta q-metering q-fwaas-v1
```

Comment:
is there conflict with the task `enable neutron services by new services names` in L14~L25 ?